Documentation/Storage-Configuration/Monitoring/ceph-dashboard.md

```diff
@@ -254,3 +254,85 @@ rook-ceph.example.com      kubernetes.io/tls   2         4m
 ```
 
 You can now browse to `https://rook-ceph.example.com/` to log into the dashboard.
+
+## Single Sign On (SSO)
+
+!!! hint
+    This is paid Koor Storage Distribution feature.
+
+This example assumes that your Ceph dashboard is currently exposed via a Kubernetes Ingress/ OpenShift Route on `https://ceph-dash.example.com` and that you are using Keycloak that is running at `https://keycloak.example.com/auth` (with a realm called `myrealm`).
+
+This is an example of the SSO config for the Ceph dashboard:
+
+```yaml
+spec:
+  dashboard:
+    enabled: true
+    # The new SSO section
+    sso:
+      enabled: true
+      baseUrl: "https://ceph-dash.example.com"
+      #entityID: "<idp_entity_id>" # optional
+      idpMetadataUrl: "https://keycloak.example.com/auth/realms/myrealm/protocol/saml/descriptor"
+      idpAttributes:
+        username: "username"
+      users:
+        - username: your_username
+          roles:
+           - administrator
+        # You can have more than one user entry
+        #- username: second_username
+        #  roles:
+        #  - rgw-manager
+```
+
+!!! note
+    You must list every user in the `users:` list before they can login to the Ceph dashboard due to how SSO is currently handled in the Ceph dashboard.
+
+Each of the fields needs to be set as follows:
+
+* `enabled` - Set to `true` to enable the dashboard SSO configuration.
+* `baseURL` - The external address your dashboard is exposed on, e.g., if you have an Ingress exposing the dashboard on `https://ceph-dash.example.com` this would be the whole URL (without a last slash added).
+* `entityID` - Optional. Use this when you have more than one entity ID in your IdP metadata.
+* `idpMetadataUrl` - Your IdP's metadata file, e.g., on Keycloak this is located here: `https://keycloak.example.com/auth/realms/myrealm/protocol/saml/descriptor` (in Keycloak you can find it under the `Realm Settings` -> `General` tab -> `SAML 2.0 Identity Provider Metadata` link).
+* `idpAttributes.username` - The SAML2 attribute name containing the user's username (optional, defaults to `uid`).
+* `users` - List of users to create and which system roles to give them. You can also specify your custom roles, but you need to have made them already.
```

Comment:
Is there a command we can add here that fetches these users and privileges, maybe something like rgw `user info`?

Reply:
Not sure if dashboard user info command would be helpful here, how about pointing to the Ceph dashboard system roles list?

Reply:
yeah I think it's a good idea to point to Ceph Dashboard roles list doc.